pkgs/build-support/wrapper-common/utils.bash

```diff
@@ -123,6 +123,7 @@ expandResponseParams() {
     local arg
     for arg in "$@"; do
         if [[ "$arg" == @* ]]; then
+            export NIX_RESPONSE_FILE_EXPANDED=1
```

Comment:
It's not clear why a new environment variable is used here instead of setting `NIX_LD_USE_RESPONSE_FILE=1`

Reply:
Oh wait I remember. It's because the cc-wrapper uses `NIX_CC_USE_RESPONSE_FILE`. I suppose the helper could export both? It feels like the utils shouldn't be tied to the wrappers like that though. What do you think?

Reply:
Or the variables could be unified under `NIX_USE_RESPONSE_FILE`?

Reply:
Actually it looks like the distinction is important on darwin where `ld64` may not support response files:
https://github.com/NixOS/nixpkgs/blob/f81112bbb0f357b80333300c4702ff7538096156/pkgs/stdenv/darwin/default.nix#L511

Reply:
... which means the logic on darwin might need to be a bit more clever?